src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -244,6 +244,7 @@ public FetchCommand fetch_() {
             public boolean shallow;
             public Integer timeout;
             public boolean tags = true;
+            public Integer depth = 1;
```

Comment:
This field will be null after xml deserialisation so you should prepare to nulls